config/puma.rb

```diff
@@ -22,3 +22,7 @@
 end
 
 plugin :tmp_restart
+
+if ENV["PROXY_PROTO_V1"]
+  set_remote_address(proxy_protocol: :v1)
+end
```

Comment:
Just a minor change to fit our code styles:
```suggestion
set_remote_address(proxy_protocol: :v1) if ENV['PROXY_PROTO_V1']
```